tasks/version.rb

```diff
@@ -39,3 +39,27 @@
     end
   end
 end
+
+task :expeditor_update_version do
```

Comment:
the whole block of green code here.  if expeditor wants to call the rake tasks and ignore errors it should just call the rake tasks and ignore errors, rather than building additional rake tasks that ignore errors...